src/ripple_app/ledger/LedgerEntrySet.cpp

```diff
@@ -1122,6 +1122,23 @@ STAmount LedgerEntrySet::rippleOwed (const uint160& uToAccountID, const uint160&
 
     if (sleRippleState)
     {
+        { // check freeze
+            
+            SLE::pointer sleIssuer = entryCache(ltACCOUNT_ROOT,
+                Ledger::getAccountRootIndex(uToAccountID));
+            if(sleIssuer)
+            {
+                if(sleIssuer->isFlag(lsfRequireAuth))
+                {
+                    if(sleRippleState->isFlag(
+                        (uFromAccountID > uToAccountID) ? lsfHighAuth : lsfLowAuth))
```

Comment:
I would just force the balance/limit to appear as 0 so that no transfer can occur - assuming the code paths are actually using those methods and not feedng off raw balances.